src/commands/highlights.rs

```diff
@@ -0,0 +1,96 @@
+use super::*;
+use crate::Arc;
+
+#[command("highlights")]
+#[sub_commands(highlights_add, highlights_get, highlights_remove)]
+#[usage("!highlights <add | get | remove>")]
+pub async fn highlights(_: &client::Context, _: &Message) -> CommandResult {
+    abort_with!(UserErr::invalid_usage(&HIGHLIGHTS_COMMAND_OPTIONS))
+}
+
+/// add a highlight for your user
+#[command("add")]
+#[usage("!highlights add <word>")]
+pub async fn highlights_add(ctx: &client::Context, msg: &Message, args: Args) -> CommandResult {
+    let args = args.message().trim().to_string();
```

Comment:
lot really a fan of just calling this `args`, as args is very explicitly the list of arguments as passed to the function